src/clients/N3Client.ts

```diff
@@ -332,9 +332,9 @@ export class N3Client {
       }),
     ]);
     const [decimalRaw, symbolRaw, nameRaw] = result;
-    let decimals: number = 0;
-    let symbol: string = "";
-    let name: string = "";
+    let decimals = 0;
+    let symbol = "";
+    let name = "";
```

Comment:
Hmm the eslint does not allow declaring of type? Maybe disable or find the specific property that prevents typing bah.

Reply:
Not sure why but adding the types no longer has an issue. Do you want me to re-add all instances where the types were removed?